Gemfile

```diff
@@ -34,6 +34,8 @@ gem 'spring',        group: :development
 
 gem 'sprockets', '>= 2.12.3'
 
+gem 'nokogiri', '>= 1.6.7.rc4'
```

Comment:
I don't think you need to pin this dependency -- `bundle update nokogiri` ought to have been good enough.